src/api/Jalasoft.TeamUp.Resumes.API/Controllers/GetResumesByName.cs

```diff
@@ -0,0 +1,54 @@
+﻿namespace Jalasoft.TeamUp.Resumes.API.Controllers
+{
+    using System.Net;
+    using Jalasoft.TeamUp.Resumes.Core.Interfaces;
+    using Jalasoft.TeamUp.Resumes.Models;
+    using Jalasoft.TeamUp.Resumes.ResumesException;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.Azure.WebJobs;
+    using Microsoft.Azure.WebJobs.Extensions.Http;
+    using Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Attributes;
+    using Microsoft.Extensions.Primitives;
+    using Microsoft.OpenApi.Models;
+
+    public class GetResumesByName
+    {
+        private readonly IResumesService resumesService;
+
+        public GetResumesByName(IResumesService resumesService)
+        {
+            this.resumesService = resumesService;
+        }
+
+        [FunctionName("GetResumesByName")]
+        [OpenApiOperation(operationId: "GetResumesByName", tags: new[] { "Resumes" })]
+        [OpenApiParameter(name: "name", In = ParameterLocation.Path, Required = true, Type = typeof(string), Description = "The name of the skill to search by.")]
+        [OpenApiResponseWithBody(statusCode: HttpStatusCode.OK, contentType: "application/json", bodyType: typeof(Resume[]), Description = "Successful response")]
+        [OpenApiResponseWithoutBody(statusCode: HttpStatusCode.NotFound, Description = "Resource not found")]
+        public IActionResult Run(
+        [HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = "v1/resumes/{name}")] HttpRequest req, string name)
```

Comment:
should the route be this?
what if there are spaces, does it work? is it following restful best practices?
does the name have to be path param or query param?